crawl-ref/source/mon-cast.cc

```diff
@@ -5189,10 +5189,12 @@ static void _cast_resonance_strike(monster &caster, mon_spell_slot, bolt&)
         return;
 
     const int constructs = _count_nearby_constructs(caster, target->pos());
-    // base damage 3d(spell hd) (probably 3d12)
+    // base damage 3d(spell hd)
     // + 1 die for every 2 adjacent constructs (so at 4 constructs, 5dhd)
```

Comment:
Should update this comment, or add an ellipsis and add another comment above the `min()` call.